python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -52,36 +53,36 @@ class ProcessCcdWithFakesConnections(PipelineTaskConnections, dimensions=("instr
     fakeCat = cT.Input(
         doc="Catalog of fake sources to draw inputs from.",
         name="{CoaddName}Coadd_fakeSourceCat",
-        storageClass="Parquet",
+        storageClass="DataFrame",
         dimensions=("tract", "skymap")
```

Comment:
@sr525, just to make sure, is it ok for the Gen 3 code to only take a tract-level `*Coadd_fakeSourceCat`, and not a detector-level `fakeSourceCat`? That seems to be the only substantial difference between the Gen 2 and Gen 3 versions of the task at the moment.

Reply:
Okay, so I get what you're asking now. The catalogs are not different in how they are stored really. Both will be at the tract level, they just have different fluxes for the visit data vs coadd data.

Reply:
Probably not? The visit level catalogues are needed for the variable sources, where the current way of using them is to make a catalogue specific to each calexp.

Reply:
Wait so are you saying we are going to produce specific catalogs for each ccdVisit vs cutting out the sources from a tract level catalog?

Reply:
Okay, this conversation doesn't seem to be continuing so I'm going to go ahead and merge the ticket.